engine/execution/computation/execution_verification_test.go

```diff
@@ -190,7 +191,7 @@ func Test_ExecutionMatchesVerification(t *testing.T) {
 		// ensure fee deduction events are emitted even though tx fails
 		require.Len(t, cr.Events[1], 3)
 		// storage limit error
-		assert.Contains(t, cr.TransactionResults[1].ErrorMessage, "Error Code: 1103")
+		assert.Contains(t, cr.TransactionResults[1].ErrorMessage, fmt.Sprintf("Error Code: %d", errors.ErrCodeInsufficientPayerBalance))
```

Comment:
errors.ErrCodeInsufficientPayerBalance.String() instead of sprintf.  same elsewhere